contents/blog/new-pricing.md

```diff
@@ -0,0 +1,144 @@
+---
+date: 2024-06-24
+title: 'PostHog is now up to 80% cheaper'
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog_is_the_cheapest_e77c4ea4a5.jpg
+author:
+  - james-hawkins
+tags:
+  - PostHog news
+---
+import {ProductScreenshot} from 'components/ProductScreenshot'
+
+export const Pricing = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_ba910c9535.jpg"
+export const PricingDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_dark_6b11387d6c.jpg"
+export const PricingComparison = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_comparison_f0c9e11f40.jpg"
+export const PricingComparisonDark = "https://res.cloudinary.com/dmukukwp6/image/upload/posthog_pricing_comparison_dark_ecf7d726c0.jpg"
+
+## TL;DR
+
+* You can now use PostHog's product and web analytics for up to 80% less, making us cheaper than any popular competitive product.
```

Comment:
```suggestion
* You can now use our product and web analytics tools for up to 80% less, making us the most cost effective solution on the market.
```

Talking about ourselves in third person and caveating competitors as referring to popularity weakens this imho

Reply:
Agree on 1, but not 100% on 2.

I think @jamesefhawkins used "cheaper" deliberately because that's what we are. "cost effective solution" feels a bit indirect and tonally wrong for us. 

If we were determined not to say "cheaper", I'd probably go for something like "the better value than any popular competitive product."